package.json

```diff
@@ -1,13 +1,18 @@
 {
   "name": "hmpo-logger",
-  "version": "7.0.2",
+  "version": "7.0.3",
```

Comment:
We don't modify the version ourselves, `np` will handle that when publishing and this will be a major version bump anyway as the ticket says